dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -2694,10 +2697,17 @@ public Contentlet fireContentWorkflow(final Contentlet contentlet, final Content
 		this.validateActionStepAndWorkflow(contentlet, dependencies.getModUser());
 		this.checkShorties (contentlet);
 
-		final WorkflowProcessor processor = this.fireWorkflowPreCheckin(contentlet, dependencies.getModUser());
+		final ThreadContextUtil contextUtil = ThreadContextUtil.getInstance();

```

Comment:
Just make this a call to a static method, I think

Reply:
Done